src/sass/components/_buttons.scss

```diff
@@ -31,7 +31,8 @@ Styleguide 2.0
   padding: 0 3rem;
   text-align: center;
   width: auto;
-
+  user-select: none;
```

Comment:
Great idea! Please just place it above width to maintain the alphabetic order and I'll go ahead and merge it :)